galvan/src/main/java/org/terracotta/testing/master/AbstractHarnessEntry.java

```diff
@@ -68,10 +68,11 @@ private boolean internalRunTestHarness(EnvironmentOptions environmentOptions, IT
     boolean wasCompleteSuccess = true;
     int clientsToCreate = master.getClientsToStart();
     for (C runConfiguration : runConfigurations) {
+      ITestStateManager stateManager = null;
```

Comment:
It is just because that commit didn't yet have the type - it was just plumbing through the interfaces.